src/main/java/com/todolist/app/service/ToDoStoreSingleton.java

```diff
@@ -1,13 +1,22 @@
 package com.todolist.app.service;
-import com.todolist.app.model.Item;
-import com.todolist.app.model.Status;
-import java.util.*;
 
 
+import com.todolist.app.model.Item;
+import com.todolist.app.model.Priority;
+import com.todolist.app.model.Status;
+import com.todolist.app.model.Tag;
+import com.todolist.app.servlet.NewTaskServlet;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
 
 
 public class ToDoStoreSingleton implements Store {
+    private Logger logger = LogManager.getLogger(ToDoStoreSingleton.class);
     private static ToDoStoreSingleton instance = null;
     private final Map<String, Item> items = new LinkedHashMap<>();
     protected ToDoStoreSingleton(){
```

Comment:
It looks like this is wrong.

```
public static ToDoStoreSingleton Instance() {
        if (instance==null) {
            instance = new ToDoStoreSingleton();
            return instance;
        }
        else
            return null;
    }
```

It should return an instance if instance is not null, e.g.

```
public static ToDoStoreSingleton Instance() {
        if (instance==null) {
            instance = new ToDoStoreSingleton();
            return instance;
        }
        else
            return instance;
    }
```